lib/gamnit/depth/depth.nit

```diff
@@ -58,6 +56,11 @@ redef class App
 			end
 		end
 
+		# Toggle writing to the depth buffer for particles effects
+		glDepthMask false
+		for system in particle_systems do system.draw
+		glDepthMask true
```

Comment:
If there is a problem with overlapping particles (in the future), we may want to change `glBlendFunc` here.